README.md

```diff
@@ -0,0 +1,10 @@
+# Koor-Tech Genesis
+
+
+Welcome to koor Genesis
+
+Our stack
+- Go
+- RabbitMQ
+- Postgres
+
```

Comment:
Let's add a section talking about the license the code is under, and let's add a license file.

Reply:
Which license should we use? MIT, or something more restrictive?

Reply:
Let's go with Apache 2.0 license